code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -328,16 +328,13 @@
 					boutput(owner, "<span class='notice'>You create a bridge on [T].</span>")
 				else
 					boutput(owner, "<span class='alert'>You were unable to place a bridge on [T].</span>")
-
-				return 1
+			return 1
 
 		var/obj/blob/B1 = T.can_blob_spread_here(owner, null, isadmin(owner))
 		if (!istype(B1))
 			return 1
-
 		if (!tutorial_check("spread", T))
 			return 1
-
```

Comment:
y tho

Reply:
Probably a mistake I thought didn't matter and thus didn't notice